Source/swiftlint/Commands/LintCommand.swift

```diff
@@ -55,7 +55,10 @@ struct LintCommand: CommandType {
             violations += currentViolations
             reporter.reportViolations(currentViolations, realtimeCondition: true)
         }.flatMap { files in
-            reporter.reportViolations(violations, realtimeCondition: false)
+            if isWarningThresholdBroken(configuration, violations: violations) {
+                violations.append(createThresholdViolation(configuration.warningThreshold!))
+            }
+            reporter.reportViolations(violations, realtimeCondition: true)
```

Comment:
Oh, I just realized that this is in `LintCommand`, which is in the `swiftlint` target, which isn't tested...

We only have unit tests set up for SwiftLintFramework.

Please disregard my comment about adding a test for this then.